test/expected/insert-17.out

```diff
@@ -372,17 +372,17 @@ EXPLAIN (costs off) INSERT INTO chunk_assert_fail SELECT i, j FROM chunk_assert_
 
 INSERT INTO chunk_assert_fail SELECT i, j FROM chunk_assert_fail WHERE i > 1;
 EXPLAIN (costs off) INSERT INTO one_space_test SELECT * FROM one_space_test WHERE time < 'infinity' LIMIT 1;
-                                                  QUERY PLAN                                                   
----------------------------------------------------------------------------------------------------------------
+                                         QUERY PLAN                                         
+--------------------------------------------------------------------------------------------
  Custom Scan (HypertableModify)
    ->  Insert on one_space_test
          ->  Custom Scan (ChunkDispatch)
                ->  Limit
                      ->  Append
-                           ->  Index Scan using _hyper_6_14_chunk_one_space_test_time_idx on _hyper_6_14_chunk
-                                 Index Cond: ("time" < 'infinity'::timestamp without time zone)
-                           ->  Index Scan using _hyper_6_15_chunk_one_space_test_time_idx on _hyper_6_15_chunk
-                                 Index Cond: ("time" < 'infinity'::timestamp without time zone)
+                           ->  Seq Scan on _hyper_6_14_chunk
```

Comment:
Not sure about this difference, maybe we should make a bigger table/disable seq scan?

We'll have to keep the versioned reference in any case though, because of the initplan explain differences.

Reply:
Hmmm... We can disable the seqscan on this test... will change and let's see what happens!!!

Reply:
Disabled seqscan globally in this regression test